arc/cli/commands/run.py

```
@@ -784,101 +787,163 @@ async def _execute_with_orchestrator(
+                # Transform raw Modal result to include success field
+                transformed_result = _transform_modal_result(update.latest_result)
+                results.append(transformed_result)
+            total_cost = update.current_cost
+
+    execution_time = time.time() - start_time
+    
+    # Calculate final metrics
+    success_count = sum(1 for r in results if r.get("success", False))
+    
+    # Create fallback results if batch processing succeeded but results array is empty
+    if not results and batch_outcomes:
```

Ruff (0.11.9)

Undefined name `batch_outcomes`

(F821)